test/reproducibility_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+bazel clean
+bazel build test:play-routes
+routes_hash0=$(md5sum bazel-bin/test/play/routes/play_routes/router/Routes.scala | awk '{print $1}')
+prefix_hash0=$(md5sum bazel-bin/test/play/routes/play_routes/router/RoutesPrefix.scala | awk '{print $1}')
+
+sleep 2
```

Comment:
Why do we need this sleep? I'm concerned that it will introduce non-determinism in the tests if we need to sleep.

Reply:
The time stamps only have second granularity; this ensures we run on different seconds. We probably don't need this though since the actions are not instantaneous and more than a few seconds will pass between them

Reply:
Ah. Makes sense. Can you please add a comment explaining that?